tests/test_midi.py

```diff
@@ -56,6 +56,14 @@ def midi_duration(level, partial):
         duration = 10
     return duration
 
+def midi_receive_with_timeout(in_port, timeout_s=10):
+    for s in range(timeout_s):
+        msg = in_port.receive(block=False)
+        is msg is not None:
```

Comment:
If

Reply:
I quickly fixed this.

Reply:
Thanks! I was coming back to that and couldn't quite work out what had happened at first..